src/main/java/service/impl/ProcessorImpl.java

```diff
@@ -0,0 +1,23 @@
+package service.impl;
+
+import java.util.List;
+import models.FruitTransaction;
+import service.Processor;
+import service.operation.OperationHandler;
+import strategy.Strategy;
+
+public class ProcessorImpl implements Processor {

```

Comment:
let's rename class as was suggested in common mistakes:
```suggestion
public class FruitServiceImpl implements FruitService {

```